src/main/java/uk/gov/hmcts/divorce/legaladvisor/event/LegalAdvisorMakeDecision.java

```diff
@@ -108,4 +114,18 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .state(endState)
             .build();
     }
+
+    public SubmittedCallbackResponse submitted(final CaseDetails<CaseData, State> details,
+                                               final CaseDetails<CaseData, State> beforeDetails) {
+
+        final CaseData caseData = details.getData();
+        final State state = details.getState();
+
+        if (caseData.getApplication().isSolicitorApplication()
+            && AwaitingClarification.equals(state)) {
+            notification.send(caseData, details.getId());
```

Comment:
adding a log would be nice including case id and which notification is sent

Reply:
Cool, will add